lib/nodejs/lib/thrift/binary_protocol.js

```diff
@@ -302,8 +302,6 @@ TBinaryProtocol.prototype.getTransport = function() {
 
 TBinaryProtocol.prototype.skip = function(type) {
```

Comment:
I wonder why this is replicated in more than one place (skip)...

Reply:
You're not alone :-)